.github/workflows/ci.yml

```diff
@@ -61,9 +61,9 @@ jobs:
         #       echo >&2 "Reproducible build broken"; cat bor1.sha256; cat bor2.sha256; exit 1
         #     fi
 
-        - name: Lint
-          if: runner.os == 'Linux'
-          run: make lint
+        # - name: Lint
+        #   if: runner.os == 'Linux'
+        #   run: make lint
```

Comment:
Why is linter disabled?

Reply:
It was inconsistent comparing to linux.

Reply:
Okay, but linter will only run when the os is linux right? Or are you saying even for linux we get inconsistent results?

Reply:
Only mac on CI was inconsistent.

Reply:
Okay, so we shouldn't comment this out right, since it is consistent on Linux?

Reply:
Yeap, this is a mistake. Could you even try to run CI linters on both OSs?

Reply:
Yes you can. It will be under two separate jobs. The code is doing the right thing, only allowing linter to run when os is linux.

Reply:
Like remove if statement from the code and restore make lint. Let's try it on both oss once again.